packages/hash/frontend/src/blocks/page/BlockContextMenu/BlockContextMenu.tsx

```diff
@@ -24,7 +24,6 @@ import { BlockEntity } from "@hashintel/hash-shared/entity";
 import { Menu, FontAwesomeIcon } from "@hashintel/hash-design-system";
 import { useUserBlocks } from "../../userBlocks";
 import { getBlockDomId } from "../BlockView";
-import { BlockSuggesterProps } from "../createSuggester/BlockSuggester";
```

Comment:
This and any other reference to `blockSuggestProps` was drilling down a function into `BlockListMenuContent` – it can get it itself via a hook, so we don't need to pass it through